tests/test_readme.cpp

```diff
@@ -11,7 +11,7 @@
 
 #include <safe/safe.h>
 
-#include <doctest/doctest.h>
+#include <catch2/catch_test_macros.hpp>
```

Comment:
Why ?

Reply:
It's the new kool kid in town. Doctest could indeed be used with fetchcontent apparently.

Do you need a rollback ?

Reply:
Since I use doctest and do not know catch, I'd rather keep my tool.